tests/integration/cattletest/core/test_volume.py

```diff
@@ -11,3 +11,73 @@ def test_volume_delete_active(client, context):
 
     volume = client.wait_success(client.delete(volume))
     assert volume.state == 'removed'
+
+
+def test_volume_create_state(client, context):
```

Comment:
Need to add tests in `test_docker.py` that the back populating from the docker inspect works.